ci-operator/config/openshift/ci-docs/openshift-ci-docs-master.yaml

```diff
@@ -39,9 +39,11 @@ tests:
       --url-ignore "$(paste -sd , <<'EOF'
     /mojo.redhat.com/
     /sippy.ci.openshift.org/
+    /github.com\/openshift\/continuous-release-jobs/
```

Comment:
> Adding continuous-release-jobs to ignore list since it's a private repo

Doc is open to all CI users.
Can we remove the link to the private repo?
You can still leave it as text, i guess.

Reply:
Sure, let me remove that, we'll just make a text reference to it and highlight that it's a private repo